Automattic/wp-calypso
# Add preferences control to dev menu

This PR adds the ability to inspect and unset [calypso preferences](http://wpcalypso.wordpress.com/devdocs/client/state/preferences/README.md) from the developer menu.

<img width="422" alt="Example" src="https://user-images.githubusercontent.com/2810519/36557901-5f197872-17be-11e8-9656-2f029df50438.png">

## Specs

Each persisted preference (ones that persist longer than a page load) should appear in a new dev menu. The value should be shown, along with the option to "unset" the preference.

## Testing Instructions
- Boot this branch locally or use the calypso.live link below
- Navigate to http://calypso.localhost:3000/devdocs/blocks/dismissible-card
- Dismiss the card labeled "I can be dismissed forever!"
- Notice that is should contain `dismissible-card-example` with a value of `true`
- Press the orange `X` to unset the preference
- Notice that the card has reappeared
